tests/helpers/contracts.py

```diff
@@ -80,7 +80,7 @@ def _instantiate(self) -> Address:
 
 class BridgeContract(LedgerContract):
     admin: Wallet = None
-    cfg: BridgeContractConfig = None
+    cfg: BridgeContractConfig
```

Comment:
BridgeContractConfig cannot be None as it's not Optional[BridgeContractConfig]. Making it Optional would require additional ifs and assertions.